external/loaders/loaders/batches/_batch.py

```diff
@@ -30,6 +39,8 @@
 logger.setLevel(logging.INFO)
 
 
+# TODO: remove this decorator or delete this function
```

Comment:
Ignored type checking on the two functions in this file because the PR I will be merging immediately after this may delete these functions.